### ISSUE TITLE
this.get('trackjs.console').log() throws TypeError this.get() is not a function

### ISSUE THREAD

**Issue by xcskier56** [jherdman/ember-cli-trackjs#76]
ember-cli-trackJs version: 2.0.0
Ember: 2.10.0

The issue is that the context in which `log()` is called is the object it was created not the service, and therefore has no access to `this.get('_isFastBoot)` 

Apologies, I accidentally submitted this before I was done writing

**Comment by jherdman** [comment]
Apologies for my extremely tardy response on this. Is a suitable workaround for you to do `this.get('trackjs').console.log('blah blah blah')`? I don't have any apps running with Fastboot right now, so it may be a bit before I can work on this.

**Comment by xcskier56** [comment]
I just ran across this issue anew when I was updating packages again. I then tried this on master and it appears that master but not the latest release fixes this. 

The culprit is/was `this` being undefined when the console was defined as a POJO in the `track.js` file. Now that the console has been extracted into its own service, this error is no more. Consider this a vote for #91 since the release will fix this.
